portlets/src/main/webapp/vue-app/engagement-center/components/common/ResultNotFound.vue

```diff
@@ -49,26 +49,79 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
         fas fa-trophy
       </v-icon>
     </div>
-    <div class="d-flex flex-column justify-center align-center py-12">
-      <span class="text-header-title font-weight-regular">
-        {{ $t('programs.details.programDeleted') }}
-      </span>
+    <div class="d-flex flex-column justify-center align-center py-12 col-sm-8 mx-auto">
+      <p class="text-header-title font-weight-regular mb-8">
+        {{ messageTitle }}
+      </p>
+      <p class="text-header-title font-weight-regular">
+        {{ messageInfoOne }}
+      </p>
+      <p 
+        v-if="sanitizedHtml" 
+        class="text-header-title font-weight-regular" 
+        v-sanitized-html="sanitizedHtml"> 
+      </p>
+      <a
+        v-if="buttonText && isButtonUrl" 
+        class="btn btn-primary my-8"
+        :href="url">
+        <span class="mx-2 text-capitalize-first-letter subtitle-1">
+          {{ buttonText }}
+        </span>
+      </a>
       <v-btn
-        class="btn btn-primary mt-8"
-        :href="programListLink">
+        v-if="buttonText && !isButtonUrl" 
```

Comment:
merge both `<a>` and `<v-btn>`
```suggestion
        v-if="buttonText" 
        :href="buttonUrl"
```